<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffManagementWindow</class>
 <widget class="QMainWindow" name="StaffManagementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="updatepushButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>更新</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="deletepushButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="returnpushButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>返回主界面</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="addpushButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
